pkg/console/operator/status.go

```diff
@@ -50,17 +50,21 @@ import (
 //
 
 const (
-	reasonUnmanaged           = "ManagementStateUnmanaged"
-	reasonRemoved             = "ManagementStateRemoved"
+	// standard reasons used by other operators for consistency
+	reasonAsExpected      = "AsExpected"
+	reasonWorkloadFailing = "WorkloadFailing"
+	// the simplest resolutions would likely come from management state
+	reasonUnmanaged = "ManagementStateUnmanaged"
+	reasonRemoved   = "ManagementStateRemoved"
+	// these are most meaningful and specific to our operator
 	reasonSyncLoopProgressing = "SyncLoopProgressing"
 	reasonNoPodsAvailable     = "NoPodsAvailable"
-	reasonSyncError           = "SyncError"
-	reasonAsExpected          = "AsExpected"
+	reasonSyncIncomplete      = "ResourceSyncIncomplete"
```

Comment:
Should this be something like `SyncInProgress` or `Reconciling`? Incomplete sounds like there's a problem.

Reply:
Sounds good, I will update.

Reply:
I will eliminate this entirely, the `reasonSyncLoopProgressing = "SyncLoopProgressing"` and `func ConditionResourceSyncProgressing()` is sufficient.